security/cmd/node_agent/na/nafactory.go

```diff
@@ -44,6 +44,14 @@ func NewNodeAgent(cfg *Config) (NodeAgent, error) {
 		return nil, err
 	}
 	na.pc = pc
-	na.cAClient = &grpc.CAGrpcClientImpl{}
+	dialOpts, err := pc.GetDialOptions()
+	if err != nil {
+		return nil, err
+	}
+	cap, err := protocol.NewGrpcConnection(cfg.CAClientConfig.CAAddress, dialOpts)
```

Comment:
The name "cap" is a little confusing, change to a better name?

Reply:
Done.